### PR TITLE
[1pt] PR: Update to README.md

### PR DESCRIPTION
**Pull request by RobHanna-NOAA**
Updated the `README.md` for the following edits:
1) Include a sample folder structure to increase clarity on download pathing.
2) Update docker run command notes, which had an error in it earlier.
3) Add some notes about downloading source code in relation to folder pathing.
4) Add more details on where to get `S3 input` data and where to save it to.
5) Updated notes on options for the calibration database system.

This PR is based on [Issue 934](https://github.com/NOAA-OWP/inundation-mapping/issues/934)

### Changes  
`README.md`: as described above

### Issuer Checklist (For developer use)

_You may update this checklist before and/or after creating the PR. If you're unsure about any of them, please ask, we're here to help! These items are what we are going to look for before merging your code._

- [x] Informative and human-readable title, using the format: `[_pt] PR: <description>`
- [x] Links are provided if this PR resolves an issue, or depends on another other PR
- [x] If submitting a PR to the `dev` branch (the default branch), you have a descriptive [Feature Branch](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow) name using the format: `dev-<description-of-change>` (e.g. `dev-revise-levee-masking`)
- [x] Changes are limited to a single goal (no scope creep)
- [x] The feature branch you're submitting as a PR is up to date (merged) with the latest `dev` branch
- [x] Changes adhere to [PEP-8 Style Guidelines](https://peps.python.org/pep-0008/)
- [x] Any _change_ in functionality is tested
- [ ] n/a - Passes all unit tests locally (inside interactive Docker container, at `/foss_fim/`, run: `pytest unit_tests/`)
- [ ] n/a - New functions are documented (with a description, list of inputs, and expected output)
- [x] Placeholder code is flagged / future todos are captured in comments
- [x] [CHANGELOG](/docs/CHANGELOG.md) updated with template version number, e.g. `4.x.x.x` 
- [x] [Reviewers requested](https://help.github.com/articles/requesting-a-pull-request-review/)
- [x] Add yourself as an [assignee](https://docs.github.com/en/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users) in the PR  as well as the FIM Technical Lead

### Merge Checklist (For Technical Lead use only)

- [ ] Update [CHANGELOG](/docs/CHANGELOG.md) with latest version number
- [ ] Update the [Citation.cff](/CITATION.cff) file to reflect the latest version number in the [CHANGELOG](/docs/CHANGELOG.md)
- [ ] If applicable, update [README](/README.md) with major alterations

### CODE REVIEW

**Comment by RobHanna-NOAA** [comment]
Adjusted the file to add info about the fim_temp docker volume mount.

File: README.md [hunk 1]
```diff
@@ -58,72 +57,107 @@ Input data can be found on the ESIP S3 Bucket (see "Accessing Data through ESIP
...
+```
+**Note**: When you include the `--dry-run` argument in the command, a large list will be returned showing you exactly which files are to be downloaded and where they will be saved. We recommend including this argument the first time you run the command, then quickly aborting it (CTRL-C) so you don't get the full list. However, you can see that your chosen target path on your machine is correct.  When you are happy with the pathing, run the `aws s3` command again and leave off the `--dry-run` argument.
+
+The S3 inputs directory has all of the folders and files you need to run FIM. It includes some publicly available and some non-publicly availible data.
+
+### Getting FIM source code
+(based on sample pathing described above)
+
+``` path to your "code" folder. e.g.
+cd /home/my_user/projects/code
```
Comment (robgpita-noaa) [inline README.md:84]: We need to insert the `fim` directory in this command to match the example folder structure listed above: `cd /home/my_user/projects/fim/code`

File: README.md [hunk 2]
```diff
@@ -58,72 +57,107 @@ Input data can be found on the ESIP S3 Bucket (see "Accessing Data through ESIP
 4. Change group ownership of repo (needs to be redone when a new file occurs in the repo):
```
Comment (robgpita-noaa) [inline README.md:57]: In Step 2 above, it is necessary to have this repository cloned locally in providing the `<path/to/repository>` . That being the case, I suggest we move the `Getting FIM source code` section above the  `Installation` section.

File: README.md [hunk 3]
```diff
@@ -58,72 +57,107 @@ Input data can be found on the ESIP S3 Bucket (see "Accessing Data through ESIP
...
+Since all of the dependencies are managed by utilizing a Docker container, we must issue the [`docker run`](https://docs.docker.com/engine/reference/commandline/run/) command to start a container as the run-time environment. The container is launched from a Docker image which was built in [Installation](#installation). The `-v <input_path>:/data` and must contain a subdirectory named `inputs` (similar to `s3://noaa-nws-owp-fim/hand_fim`). If the pathing is set correctly, we do not need to adjust the `params_template.env` file, and can use the default file paths provided.
```
Comment (robgpita-noaa) [inline README.md:108]: "The `-v <input_path>:/data`  must contain" reads more smoothly than  

"The `-v <input_path>:/data` and must contain"

**Review by robgpita-noaa — COMMENTED** [review]
This PR looks good! I have a couple small comments/suggestions while we're at it.

**Review by BradfordBates-NOAA — APPROVED** [review]

**Pull request merged**